### ISSUE TITLE
Toc2 includes the notebook's title in the toc

### ISSUE THREAD

**Issue by NicolasWoloszko** [ipython-contrib/jupyter_contrib_nbextensions#809]
...and let the title be 1. in the numerotation, so the first subtitle is 2....

**Comment by jfbercher** [comment]
I do not understand what you mean. 
Please give an example.

**Comment by NicolasWoloszko** [comment]
![capture d ecran 2016-12-01 a 11 43 45](https://cloud.githubusercontent.com/assets/12294532/20791068/8d2088aa-b7bb-11e6-8082-070c045ab8bd.png)

As you see in this screenshot, the title of the notebook (Estimation of panel regressions with time-invariant variables) appears on top of the TOC. Moreover, it is centered because the toc includes the <center> tag. One does not want the title of a notebook (which is the very first markdown header) to appear in the toc it makes no sense.

**Comment by jfbercher** [comment]
Ok, I guess that you have entered your title as a markdown header, eg 
```
# something
```
or 
``` 
<H1> Something </H1> 
```
The toc2 extension collects all headers in the notebook to build the table of contents and has no mean to distinguish what you consider as a title. Sometimes the static toc  is to be placed at the very beginning of the notebook, sometimes at the end, or elsewhere. In this version, if you want to add a title which is not numbered and does not appear in toc, use standard html in a markdown cell, as eg 
```
<p style="margin-top: 22px; font-size:40px;  color:blue; text-align: center;  
font-weight: bold;  line-height: 100%;"> Your Title here </p>
```

**Comment by jfbercher** [comment]
Hopefully fixed by #810
